lib/mocha.js

```diff
@@ -759,6 +761,7 @@ Mocha.prototype.run = function(fn) {
   var options = this.options;
   options.files = this.files;
   var runner = new exports.Runner(suite, options.delay);
+  createStatsCollector(runner);
```

Comment:
No issue with moving it here, but could you explain the problem it caused where it was.

Reply:
Circular dependency.  Runner would require stats-collector which would in turn require runner.  Can create problems with various tools (browserify, test runners, etc)